content/info/awards/best-paper-awards.md

```diff
@@ -55,7 +55,7 @@ Authors: Fritz Lekschas, Xinyi Zhou, Wei Chen, Nils Gehlenborg, Benjamin Bach, a
 
 **Data Visceralization: Enabling Deeper Understanding of Data Using Virtual Reality**
 <br/>
-Authors: Benjamin, David Brown, Bongshin Lee, Christophe Hurter, Steven Drucker, and Tim Dwyer 
+Authors: Benjamin Lee, David Brown, Bongshin Lee, Christophe Hurter, Steven Drucker, and Tim Dwyer 
```

Comment:
I thought it was "Bach"?  I might've misheard in the meeting

Reply:
It's Lee, I checked. Thanks though (definitely good to make sure I don't screw stuff up!)